test/compilable/b19775.d

```diff
@@ -0,0 +1,14 @@
+// https://issues.dlang.org/show_bug.cgi?id=19775
+enum x1 = 42;
+enum ident(alias a, args...) = mixin(a, args);
+
+//enum x2 = ident!("x1"); //FIXME - empty args
+enum x2 = x1;
```

Comment:
For now I've given up on getting an empty sequence to be ignored by `mixin()`. It still gets turned into `"()"`.